src/utils/SConscript

```diff
@@ -4,16 +4,19 @@ import daos_build
 def scons():
     """Execute build"""
     Import('env', 'prereqs')
-
-    libs = ['daos', 'gurt', 'cart', 'daos_common', 'uuid']
+    libs = ['daos', 'gurt', 'cart', 'daos_common', 'uuid', 'dfs']
```

Comment:
you don't need duns here? i think you do..

Reply:
Done